src/main/java/frc/robot/RobotContainer.java

```diff
@@ -65,6 +65,7 @@ public RobotContainer() {
   private void configureButtonBindings() {
     OI.Driver.getOrientationButton().onTrue(new InstantCommand(swerveSubsystem::toggleOrientation));
     OI.Driver.getZeroButton().onTrue(new InstantCommand(swerveSubsystem::zeroHeading));
+    OI.Driver.getParkButton().whileTrue(new InstantCommand(swerveSubsystem::parkModules,swerveSubsystem));
```

Comment:
whileTrue would require the driver to hold the button to keep it parked. That might work, but toggleOnTrue might be better. Get input from our drivers

Reply:
Zane said that it would be better to leave it as a whileTrue because if it was on for some reason due to it being a toggleOnTrue than we would think our robot isn't moving for no reason